.teamcity/src/main/kotlin/promotion/PromotionProject.kt

```diff
@@ -39,6 +39,7 @@ class PromotionProject(branch: VersionedSettingsBranch) : Project({
         param("env.ORG_GRADLE_PROJECT_sdkmanKey", "8ed1a771bc236c287ad93c699bfdd2d7")
         param("env.PGP_SIGNING_KEY", "%pgpSigningKey%")
         param("env.PGP_SIGNING_KEY_PASSPHRASE", "%pgpSigningPassphrase%")
+        password("env.GRADLE_ENTERPRISE_ACCESS_KEY", "%ge.gradle.org.access.key%;%e.grdev.net.access.key%")
```

Comment:
We don't need this here, this is already set on the individual build configurations, right? See `BasePromotionBuildType`.